Wordle.Domain/GuessStatistics.cs

```diff
@@ -0,0 +1,16 @@
+namespace Wordle.Domain;
+
+public class GuessStatistics
+{
+    public WordScore[] GuessArray = new WordScore[6];
```

Comment:
The name `GuessArray` exposes implementation.  What if this changes into some kind of new `Class` or becomes a `List` or a `Dictionary`?  Is there a better name that would expose intent?

Reply:
Secondly, should these be public variables?  It would be more secure to have private variables, and then have class methods that provide command / query separation.